spec/tapioca/compilers/sorbet_spec.rb

```diff
@@ -0,0 +1,30 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "spec_helper"
+
+class Tapioca::Compilers::SorbetSpec < Minitest::Spec
+  before do
+    @temp_env_value = T.let(nil, T.nilable(String))
+    @temp_env_value = ENV["TPC_SORBET_EXE"]
+  end
+
+  after do
+    ENV["TPC_SORBET_EXE"] = @temp_env_value
+  end
```

Comment:
I'd rather use a helper method instead of changing test state:
```ruby
def with_custom_sorbet_exe_path(path, &block)
  sorbet_exe_env_value = ENV["TAPIOCA_SORBET_EXE"]
  begin
    ENV["TAPIOCA_SORBET_EXE"] = path
    block.call(path)
  ensure
    ENV["TAPIOCA_SORBET_EXE"] = sorbet_exe_env_value
  end
end
```
and then use it in tests like:
```ruby
with_custom_sorbet_exe_path("custom_value") do |custom_path|
  assert_equal(Tapioca::Compilers::Sorbet.sorbet_path, custom_path)
end
```
This way, the test is way more readable, you don't need any cleanup steps and reduce the number of local variables you have to use in tests.

Reply:
This is great, thanks for the suggestion!